packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2725,6 +2743,20 @@ export class ContainerRuntime
 			throw e;
 		}
 	}
+
+	/**
+	 * Process an empty batch, which will execute expected actions while processing even if there are no messages.
+	 * This is a separate function because the processCore function expects at least one message to process.
+	 */
+	private processEmptyBatch(sequenceNumber) {
```

Comment:
Pulling this into a function is good, goes along very harmoniously with @pragya91's change